### ISSUE TITLE
`millisecond` conversion

### ISSUE THREAD

**Issue by joaomcteixeira** [MDAnalysis/mdanalysis#3474]
```
  File "/home/joao/anaconda3/envs/taurenmddev/lib/python3.8/site-packages/MDAnalysis/units.py", line 363, in convert
    raise ValueError(errmsg) from None
ValueError: unit 'ms' not recognized.
It must be one of Angstrom, A, angstrom, Å, nm, nanometer, pm, picometer, fm, femtometer, Angstrom^{-3}, A^{-3}, Å^{-3}, nm^{-3}, nanometer^{-3}, Molar, SPC, TIP3P, TIP4P, water, ps, picosecond, fs, femtosecond, ns, nanosecond, second, sec, s, AKMA, e, Amber, C, As, Angstrom/ps, A/ps, Å/ps, Angstrom/picosecond, angstrom/picosecond, Angstrom/fs, Angstrom/femtosecond, angstrom/femtosecond, angstrom/fs, Angstrom/AKMA, nm/ps, nanometer/ps, nanometer/picosecond, nm/ns, pm/ps, m/s, kJ/(mol*Angstrom), kJ/(mol*A), kJ/(mol*Å), kJ/(mol*nm), Newton, N, J/m, kcal/(mol*Angstrom), kJ/mol, kcal/mol, J, eV.
```

Would it be interesting to implement `millisecond` (`ms`) conversion in `MDAnalaysis.units.py` ?

**Comment by orbeckst** [comment]
sure, we're getting there — add it ;-)

**Comment by Atharva7K** [comment]
Can I work on this?

**Comment by richardjgowers** [comment]
Yes definitely, that would be fantastic!

**Comment by joaomcteixeira** [comment]
Thanks @Atharva7K :+1: would be great to see this implemented.

**Comment by pranjii** [comment]
I have made the required changes but am waiting for my last pull request to be approved

**Comment by IAlibay** [comment]
@Atharva7K @pranjii - MDAnalysis doesn't usually assign specific issues to folks (unless it's a pre-existing project, or a larger issue), we generally work on a "first PR open" basis, so if you have a fix for this please do go ahead and open it. That being said - we do generally try to strongly encourage a positive community spirit, so we would ask folks to work together & communicate with each other with the aim of not duplicating efforts.

**Comment by orbeckst** [comment]
For anyone coming to this issue and wondering if they can work on it: 

1. Read https://github.com/MDAnalysis/mdanalysis/issues/3474#issuecomment-991019927 . 
2. Then create a pull request *that references issue `#3474` (this issue) in its summary* (the template that pops up should have as the first line `Fixes #` and you complete it to read `Fixes #3474`). This will make the PR show up *here* and then everybody can see what's been worked on. You have to create a PR, just working on a fork is not sufficient for us to recognize that you are working on a specific issue. 

We will generally prioritize reviewing the first PR that shows up but if we don't get any or slow responses to comments we will move to the next PR.

At the writing of this comment, there is *no PR open*.
